packages/joget/joget/upstream.yaml

```diff
@@ -0,0 +1,4 @@
+HelmRepo: https://jogetworkflow.github.io/helm-joget/charts
+HelmChart: joget
+Vendor: joget
```

Comment:
Do you want to display `Joget` (i.e. with a capital J) as the vendor name in the Rancher UI? If so, you should have `Vendor: Joget`. Otherwise there is no point in setting `Vendor: joget`, since `joget` can be gotten from the package path.